Style-Guide/Code-Layout-and-Formatting.md

```diff
@@ -82,7 +82,7 @@ enum Color {
 function Test-Code {
     [CmdletBinding()]
     param(
-        [int]$ParameterOne
+        [Int]$ParameterOne
```

Comment:
Lowercase is correct here. `[int]` is a TypeAccelerator for `[System.Int32]` and the TypeAccelerators in PowerShell mimic C# in that they are lowercase (and our rule for capitalization is to follow what's in the shell and shows up for tab-completion). I.e. try this: 

```posh
[psobject].assembly.gettype("System.Management.Automation.TypeAccelerators")::Get
```

Reply:
Does this mean that the TypeAccelerators in this example should change then?
```PowerShell
function Write-Host {
    <#
    .SYNOPSIS
    Writes customized output to a host.
    .DESCRIPTION
    The Write-Host cmdlet customizes output. You can specify the color of text by using
    the ForegroundColor parameter, and you can specify the background color by using the
    BackgroundColor parameter. The Separator parameter lets you specify a string to use to
    separate displayed objects. The particular result depends on the program that is
    hosting Windows PowerShell.
    #>
    [CmdletBinding()]
    param(
        [Parameter(Position = 0, ValueFromPipeline = $true, ValueFromRemainingArguments = $true)]
        [PSObject]
        $Object,

        [Switch]
        $NoNewline,

        [PSObject]
        $Separator,

        [System.ConsoleColor]
        $ForegroundColor,

        [System.ConsoleColor]
        $BackgroundColor
    )
    begin {
    ...
```